tsconfig.lint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+  "include": ["./src/**/*.ts", "./test/**/*.ts"],
+  "compilerOptions": {
+    // Expand the rootDir to include the test directory
```

Comment:
Is this a syntax error to include a comment inside a JSON?

Reply:
Nope! TSC reads [JSON5](https://json5.org/) which can allow comments 🙂